evm/rpc/main.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+from evm.rpc.modules import Eth
+
+
+def generate_response(request):
+    return {
+        'id': request['id'],
+        'jsonrpc': request['jsonrpc'],
+    }
+
+
+class RPCServer:
```

Comment:
We want our RPC server to be asyncio-compatible since that's our approach to concurrency. The current `Chain` class doesn't do concurrency but eventually it will, just like `LightChain`. The existing one in rpc/server.py could be used as an example

Reply:
I started to think about how to be compatible with the current `Chain` and the future async chain. I didn't come up with anything good, but I don't know if that's because I'm an `asyncio` n00b or if there just isn't an elegant solution.

My current best idea is to go through and make all of the relevant `Chain` methods `async def` without any other changes. (I suppose I would also have to make any current Chain callers asyncio aware, too). Then I can start writing the server against a unified and future-proof async `Chain` interface.

Do you see any obvious problems with going this direction?

Reply:
@gsalgado can likely give a more definitive answer, but this seems at least worth spending a few hours on to validate it's viability.

Reply:
Yup, I think so, too. Starting to dig in...

Reply:
Well, I believe that once we have a full client syncing blocks, it won't make sense to run the JSON-RPC with a LightChain, right? If that's the case, we can have this initial RPC server asyncio-compatible (by using aiohttp), and using the `await` keyword when calling `LightChain` methods, but when we switch it to using `Chain` we'd simply drop the `await` keyword in method calls